performance/do-pings.php

```diff
@@ -16,3 +16,13 @@ function disable_pings( $event ) {
 	return $event;
 }
 add_action( 'schedule_event', __NAMESPACE__ . '\disable_pings' );
+
+// Don't allow new _encloseme metas
+function block_encloseme_metadata_filter( $should_update, $object_id, $meta_key, $meta_value, $unique ) {
+	if ( '_encloseme' === $meta_key ) {
+		$should_update = false;
```

Comment:
Noting that `add_metadata()` will return this value, so the calling code will ultimately receive `false`. Just want to be sure that's accounted for with the `_encloseme` stuff - will it perpetually keep trying to add it or similar, if it receives `false`?